rust-toolchain.toml

```diff
@@ -1,3 +1,5 @@
 [toolchain]
 channel = "nightly-2022-10-30"
 targets = [ "wasm32-unknown-unknown" ]
+components = [ "rustfmt", "clippy", "rust-src" ]
```

Comment:
can we specify different channels for build and different for lints&check :crossed_fingers:? If so we can use stable for builds :D